### ISSUE TITLE
Broken Links due to Moved Files 

### ISSUE THREAD

**Issue by meaghanfitzgerald** [ava-labs/avalanche-docs#1653]
**Describe the bug**

The CLI files moved to new folders in [this PR](https://github.com/ava-labs/avalanche-docs/pull/1648/files) were not accounted for in the `_redirects` file. This has resulted in many **broken links.**

**To Reproduce**

running `yarn build` in the terminal will reveal the following: 

```
[WARNING] Docusaurus found broken anchors!

Please check the pages of your site in the list below, and make sure you don't reference any anchor that does not exist.
Note: it's possible to ignore broken anchors with the 'onBrokenAnchors' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken anchors found:
- Broken anchor on source page path = /build/cross-chain/awm/overview:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
- Broken anchor on source page path = /build/cross-chain/teleporter/overview:
   -> linking to #teleporter-protocol (resolved as: /build/cross-chain/teleporter/overview#teleporter-protocol)
- Broken anchor on source page path = /build/subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
- Broken anchor on source page path = /build/subnet/deploy/fuji-testnet-subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
   -> linking to /nodes/run/node-manually#connect-to-fuji-testnet
   -> linking to /nodes/validate/what-is-staking#fuji-testnet
   -> linking to /build/subnet/deploy/local-subnet#create-a-custom-subnet-configuration
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /build/subnet/deploy/mainnet-subnet:
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /build/subnet/info/wagmi:
   -> linking to /build/subnet/info/wagmi#adding-wagmi-to-metamask
- Broken anchor on source page path = /build/subnet/upgrade/considerations-subnet-upgrade:
   -> linking to /nodes/configure/avalanchego-config-flags#snow-sample-size-int
- Broken anchor on source page path = /build/subnet/upgrade/customize-a-subnet:
   -> linking to /reference/subnet-evm/api#ethgetchainconfig
- Broken anchor on source page path = /build/vm/create/rust-vm:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /build/vm/golang-vms/golang-vm-simple:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /build/vm/rust-vms/installing-vm:
   -> linking to /nodes/configure/avalanchego-config-flags#vm-configs
- Broken anchor on source page path = /deprecated/tutorials-contest/2022/local-subnet-development:
   -> linking to /build/subnet/utility/deploy-smart-contract-to-subnet#step-1-setting-up-metamask
- Broken anchor on source page path = /learn/avalanche/subnets-overview:
   -> linking to /nodes/run/node-manually#requirements
- Broken anchor on source page path = /nodes/maintain/setting-up-node-monitoring:
   -> linking to /nodes/run/third-party/aws-node#f8df
- Broken anchor on source page path = /nodes/run/third-party/google-cloud-node:
   -> linking to #setup-instructions (resolved as: /nodes/run/third-party/google-cloud-node#setup-instructions)
- Broken anchor on source page path = /nodes/validate/what-is-staking:
   -> linking to /intro#avax
- Broken anchor on source page path = /reference/avalanchego/admin-api:
   -> linking to /build/vm/intro#load-a-vm
- Broken anchor on source page path = /reference/avalanchego/p-chain/txn-format:
   -> linking to /reference/standards/cryptographic-primitives#secp-256-k1-addresses
- Broken anchor on source page path = /reference/avalanchego/x-chain/api:
   -> linking to /learn/avalanche/avalanche-platform#exchange-chain-x-chain
- Broken anchor on source page path = /reference/avalanchego/x-chain/txn-format:
   -> linking to /reference/standards/cryptographic-primitives#secp-256-k1-addresses
- Broken anchor on source page path = /reference/standards/guides/issuing-api-calls:
   -> linking to /nodes/configure/chain-config-flags#--http-host-string
- Broken anchor on source page path = /reference/standards/guides/x-chain-migration:
   -> linking to /reference/avalanchego/index-api#x-chain-vertices
- Broken anchor on source page path = /tooling/avalanchejs-overview:
   -> linking to /#Requirements
- Broken anchor on source page path = /tooling/cli-create-deploy-subnets/deploy-fuji-testnet-subnet:
   -> linking to /learn/avalanche/subnets-overview#virtual-machines
   -> linking to /nodes/run/node-manually#connect-to-fuji-testnet
   -> linking to /nodes/validate/what-is-staking#fuji-testnet
   -> linking to /build/subnet/deploy/local-subnet#create-a-custom-subnet-configuration
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
- Broken anchor on source page path = /tooling/cli-create-deploy-subnets/deploy-mainnet-subnet:
   -> linking to /nodes/validate/add-a-validator#add-a-validator-with-avalanche-wallet
```

@laviniat1996
[Action Required:] **Please fix these broken links with a New PR**

**Comment by meaghanfitzgerald** [comment]
A nit: 

The new folders created in that PR should be nested into 1 CLI folder instead of having 3 CLI folders at the same level as the other tools. 

![image](https://github.com/ava-labs/avalanche-docs/assets/99047250/20633c51-a832-40d7-941f-2b833ed74e77)

**Comment by meaghanfitzgerald** [comment]
cc: @sukantoraymond

**Comment by meaghanfitzgerald** [comment]
@sukantoraymond 
when a file is moved to a new path, all links to that file are broken. **the links in all other docs must therefore be adjusted, and all new file paths included in the `_redirects` file.** 

the compartmentalization of tutorials makes sense, but _it is just my opinion_ that the folder structure is _not great_. 

i would've done:
```
├── docs
│   └── tooling
│   │   └── cli
│   │   │   └── guides
│   │   │   └── nodes
│   │   │   └── subnets
```

**Comment by laviniat1996** [comment]
closing this as it was fixed by #1666 , there are still issues with the spanish docs links, but that's another issue to tackle
